doc/everest_api_specs/external_energy_limits_consumer_API/asyncapi.yaml

```
@@ -240,6 +240,12 @@ components:
       name: receive_heartbeat
       title: 'Receive heartbeat'
       summary: Heartbeat produced by EVerest as configured via cfg_heartbeat_interval_ms in the EVerest configuration
```

```suggestion
      summary: Heartbeat produced by EVerest as configured via cfg_heartbeat_interval_ms in the EVerest configuration
      contentType: application/json
```
Not introduced by this PR, but still missing.